<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Segment</class>
 <widget class="QWidget" name="Segment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>More pages</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>More 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>More 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>More 3</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="buttonPage1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>page 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonPage2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>page 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBackMain">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>260</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Back to Main</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="buttonBack">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>107</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Page 1</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="buttonBack2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Page 2</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
